src/pds/registrysweepers/ancestry/generation.py

```diff
@@ -176,7 +176,7 @@ def get_nonaggregate_ancestry_records(
                 'Failed to parse collection and/or product LIDVIDs from document in index "%s" with id "%s" due to %s: %s',
                 doc.get("_index"),
                 doc.get("_id"),
-                type(err),
+                type(err).__name__,
```

Comment:
Nothing to change; I just thought this was neat. I usually do `err.__class__.__name__` but that's a lot of dunderuff.